resources/authgear/templates/en/translation.json

```diff
@@ -1079,12 +1079,12 @@
   "v2-csrf-error-instruction-ios-step1": "Go to the <b>Settings</b> app",
   "v2-csrf-error-instruction-ios-step2": "Go to <b>Safari &rarr; Advanced</b>",
   "v2-csrf-error-instruction-ios-step3": "Turn Off <b>Block All Cookies</b>",
-  "v2-csrf-error-instruction-chrome-desktop-step1": "At the top right, click More \"<img src=\"{settingIcon}\" alt=\"\" height=\"26\" width=\"26\" class=\"inline align-middle\"/>\" &rarr; Settings. Or visit chrome://<wbr>settings/<wbr>content/<wbr>siteData",
+  "v2-csrf-error-instruction-chrome-desktop-step1": "At the top right, click More \"<img src=\"data:image/png;base64,{settingIcon}\" alt=\"\" height=\"26\" width=\"26\" class=\"inline align-middle\"/>\" &rarr; Settings. Or visit chrome://<wbr>settings/<wbr>content/<wbr>siteData",
```

Comment:
`At the top right, click More <i class="csrf-instruction-icon icon-settings"></i> Settings. ......`